Documentation Standard Updates (#6)
fivetran/dbt_sap: README.md

    - Brings in general ledger models like General Ledger: Balances, Leading Ledger (`sap__0fi_gl_10`) and Line Items Leading Ledger (`sap__0fi_gl_14`).
    - Brings in master text models like Company Code (`sap__0comp_code_text`), Company (`sap__0company_text`), and Vendor (`sap__0vendor_text`).
- Produces modeled tables that leverage SAP data from [Fivetran's SAP connectors, like LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements), [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent) or [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana) and build off the output of our [SAP source package](https://github.com/fivetran/dbt_sap_source).
- Generates a comprehensive data dictionary of your source and modeled sap data through the [dbt docs site](https://fivetran.github.io/dbt_sap/).

> The following table provides a detailed list of all models materialized within this package by default.
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_sap/#!/overview?g_v=1).
> 
> | **model**                         | **description**                                                                                                                                                                                                                             |

Author: ```suggestion
<!--section=“sap_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_sap/#!/overview?g_v=1).

| **Table**                         | **Description**                                                                                                                                                                                                                             |
```

| [sap__0fi_gl_14](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0fi_gl_14)  | This model extracts line items from the leading ledger in new General Ledger Accounting.                                                                |
| [sap__0gl_account_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0gl_account_attr)    |   This model is used for loading G/L Account Number master data, originating from the ska1 source.                 |
| [sap__0material_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0material_attr)    |  This model is used to display material attribute information, originating from the mara source.                                                           |
| [sap__0vendor_attr](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_attr)    |  This model is used to display vendor attributes, originating from the lfa1 source.                           |
| [sap__0vendor_text](https://fivetran.github.io/dbt_sap/#!/model/model.sap.sap__0vendor_text) |  This model is used to display vendor text, originating from the lfa1 source.     |

> ## How do I use the dbt package?

Author: ```suggestion
<!--section-end-->

## How do I use the dbt package?
```

### Step 1: Prerequisites
To use this dbt package, you must have the following:
- At least one Fivetran of the following SAP connectors:

> - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
> - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
> - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)

Author: ```suggestion
   - [LDP SAP Netweaver](https://fivetran.com/docs/local-data-processing/requirements/source-and-target-requirements/sap-netweaver-requirements)
   - [HVA SAP](https://fivetran.com/docs/databases/sap-erp/high-volume-agent)
   - [SAP ERP on HANA](https://fivetran.com/docs/databases/sap-erp/sap-erp-hana)
```

- Within the connector, syncing the following respective tables into your destination:

> - bkpf
> - bseg
> - faglflexa
> - faglflext
> - kna1
> - lfa1
> - mara
> - pa0000
> - pa0001
> - pa0007
> - pa0008
> - pa0031
> - ska1
> - t001
> - t503
> - t880

Author: ```suggestion
   - bkpf
   - bseg
   - faglflexa
   - faglflext
   - kna1
   - lfa1
   - mara
   - pa0000
   - pa0001
   - pa0007
   - pa0008
   - pa0031
   - ska1
   - t001
   - t503
   - t880
```
